src/components/Input/index.tsx

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
 
-import SimpleInput from '@/components/Input/SimpleInput';
+import DefaultInput from '@/components/Input/DefaultInput';
```

Comment:
We can merge the DefaultInput and the Input component. They're the same component, so it's not necessary to hide the input's implementation in another component